src/components/layouts/MainLayout.jsx

```diff
@@ -41,7 +43,12 @@ export const MainLayout = () => {
     desktopValue: 18,
   })
 
-  const handleThemeChange = () => setThemeType(themeType === 'light' ? 'dark' : 'light')
+  const handleThemeChange = () => {
+    const newTheme = themeType === 'light' ? 'dark' : 'light'
+    console.log(newTheme)
+    updateTheme({ theme: newTheme })
```

Comment:
Would be better to leave previous function and move this logic to theme context.